lib/dl_core/dl_core_tests/conftest.py

```diff
@@ -1,11 +1,15 @@
+from aiohttp.pytest_plugin import aiohttp_client
 import pytest
 
 from dl_testing.utils import get_root_certificates
 
 
-pytest_plugins = ("aiohttp.pytest_plugin",)  # and it, in turn, includes 'pytest_asyncio.plugin'
```

Comment:
@MCPN do i remember right that last time this change broke in private that's why it was reverted?